.goreleaser.yml

```diff
@@ -0,0 +1,25 @@
+# This is an example goreleaser.yaml file with some sane defaults.
```

Comment:
Is it really just the example defaults ? Are there some modifications to be made ?

Reply:
This is the result of the command `goreleaser init`. :) With this configuration, it will only produce the binaries and attach them to the release on the assets section here: https://github.com/tiramiseb/terraform-provider-gandi/releases

If you need more formats (like a debian package, or a Homebrew receipe), you'll need to customize it a bit.